tests/layer_tests/pytorch_tests/test_select.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('input_dim', list(range(4)))
```

Comment:
Can dim and index be negative? If yes, please add tests

Reply:
Right, it can be a negative number! Good catch, thanks for this comment :) I added proper test data to the test.